plugins/wysiwygarea/plugin.js

```diff
@@ -93,8 +93,44 @@
 						return;
 					}
 
-					editor.editable( new framedWysiwyg( editor, iframe.$.contentWindow.document.body ) );
+					editor.editable( new framedWysiwyg( editor, iframe.getFrameDocument().getBody() ) );
 					editor.setData( editor.getData( 1 ), callback );
+
+					iframe.$.onloadFromSetData = false;
+
+					editor.on( 'mode', backupOnLoad, { iframe: iframe, editor: editor, callback: callback } );
```

Comment:
For the future reference:
At the first glance it looks cumbersome, but I think it has to stay this way 🤔 
If we just call `backupOnLoad()` instead of adding listener, it breaks on toggling source mode.
If we don't pass the context, editor breaks after second toggle.
We could pass `this` instead, but we would need to dig for the other necessary constructs, so it seems pointless.